document/core/appendix/algorithm.rst

```diff
@@ -122,9 +123,16 @@ The control stack is likewise manipulated through auxiliary functions:
      ctrls.pop()
      return frame
 
-   func label_types(frame : ctrl_frame) : list(val_types) =
+   func label_val_types(frame : ctrl_frame) : list(val_type) =
      return (if frame.opcode == loop then frame.start_types else frame.end_types)
 
+   func is_catch(frame : ctrl_frame) : bool =
+     return frame.catch_label
```

Comment:
```suggestion
     return frame.opcode = catch || frame.opcode = catch_all
```
We could also simply inline this condition. But this function does not currently seem to be used anywhere?